charts/domain-server/templates/statefulset.yaml

```diff
@@ -80,6 +74,10 @@ spec:
                   key: {{ $secretKey }}
             {{- end }}
           {{- end }}
+          volumeMounts:
+            - name: private-key-volume
+              mountPath: /private-key-volume
```

Comment:
I think I tried this before but it didn't work unless I used `subPath` like in Hagall: https://github.com/aukilabs/helm-charts/blob/0ee7a86710efada05f8c5af5bcc58ab2476b8374/charts/hagall/templates/deployment.yaml#L56-L59

Reply:
hmm, I think it is because you projected the whole secret. but here I only project `wallet_private.key`, it should be fine. But I added it anyway